server/cluster/cluster.proto

```diff
@@ -29,6 +29,7 @@ message ClusterCreateFromKubeConfigRequest {
 	string context = 2;
 	bool upsert = 3;
 	bool inCluster = 4;
+	string systemNamespace = 5;
```

Comment:
This is part of the public API of Argo CD and therefore for needs to be documented (yes, I know the other values aren't).

Can you please add a comment to this the run `make codegen`?  It'll then update `swagger.json` automatically.

Otherwise looks good to me.

Reply:
Fixed with latest commit.